docs/DeveloperGuide.md

```diff
@@ -287,28 +287,88 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: List all TA applicants**
```

Comment:
Charmaine this part, could do this instead:

You might need to add a new commit :') I let you decide!

```
1.  User requests to list TAs
2.  TAlent Assistant™ shows all TAs

    Use case ends.

**Extensions**

* 2a. The list is empty.

    * 2a1. TAlent Assistant™ feedbacks no TAs in system

      Use case ends.
```

Reply:
Have made and committed the change! Thanks for the suggestion!